provision.sh

```diff
@@ -7,10 +7,16 @@ echo "---------------------------------------------"
 sudo apt-get update
```

Comment:
I have a problem with all the `sudo`. I checked some other provisioners. They don't use `sudo`. Do we need `sudo`?